src/pybind/mgr/cephadm/templates/services/nvmeof/ceph-nvmeof.conf.j2

```diff
@@ -41,10 +41,10 @@ config_file = /etc/ceph/ceph.conf
 id = {{ rados_id }}
 
 [mtls]
-server_key = {{ spec.server_key }}
-client_key = {{ spec.client_key }}
-server_cert = {{ spec.server_cert }}
-client_cert = {{ spec.client_cert }}
+server_key = /server.key
+client_key = /client.key
+server_cert = /server.cert
+client_cert = /client.cert
```

Comment:
The fact that this config file is shared both for clients and servers makes it harder, but this should be something like:
```suggestion
# server
server_key = /server.key
server_cert = /server.cert
root_ca_cert = /root_ca.cert
```

```suggestion
# client
client_key = /server.key
client_cert = /server.cert
root_ca_cert = /root_ca.cert
```

Reply:
I'm a bit confused as to whether you mean we need a single root CA cert to verify the client/server certs or we want two separate CA certs, one for client, one for server